src/diffusers/models/unets/unet_motion_model.py

```diff
@@ -394,26 +1480,47 @@ def __init__(
             output_channel = block_out_channels[i]
             is_final_block = i == len(block_out_channels) - 1
 
-            down_block = get_down_block(
-                down_block_type,
-                num_layers=layers_per_block[i],
-                in_channels=input_channel,
-                out_channels=output_channel,
-                temb_channels=time_embed_dim,
-                add_downsample=not is_final_block,
-                resnet_eps=norm_eps,
-                resnet_act_fn=act_fn,
-                resnet_groups=norm_num_groups,
-                cross_attention_dim=cross_attention_dim[i],
-                num_attention_heads=num_attention_heads[i],
-                downsample_padding=downsample_padding,
-                use_linear_projection=use_linear_projection,
-                dual_cross_attention=False,
-                temporal_num_attention_heads=motion_num_attention_heads[i],
-                temporal_max_seq_length=motion_max_seq_length,
-                transformer_layers_per_block=transformer_layers_per_block[i],
-                temporal_transformer_layers_per_block=temporal_transformer_layers_per_block[i],
-            )
+            # TODO(aryan): Can we reduce LOC here by creating a dictionary of common arguments and then passing **kwargs?
```

Comment:
@DN6 WDYT about this?

Reply:
Could you give me an example here?

Reply:
Yes sure, let's roll with what we have right now for arguments. Can take this up later and look for suggestions